docs/releases.md

```diff
@@ -6,11 +6,81 @@ sidebar_label: Releases
+  * Raw block volume support
+  * Snapshot support
+  * Ability to schedule based on the capacity of the volumes provisioned
+  * Ensure that LVM volume create and deletion functions are idempotent
+- NDM partition discovery was updated to fetch the device details from it parent block device.
+
+
+### Key Bug Fixes
+- Fixed an issue with cStor helm chart w.r.t to CVC operator service that was causing velero restores to fail. (https://github.com/openebs/cstor-operators/pull/239)
+- Fixed an issue with cStor CSPC validation causing failures in creating pools with ZFS supported compression algorithms. (https://github.com/openebs/cstor-operators/issues/237)
+- Fixed an issue with the cStor validation webhook not getting deleted in K8s 1.20+ when the OpenEBS namespace is deleted. https://github.com/openebs/openebs/issues/3338)
+- Fixed an issue with Jiva replicas attempting duplicate registration if the Pod IP of the replica changes. (https://github.com/openebs/openebs/issues/3323)
+- Fixed an issue where Jiva cleanup jobs can't be executed when cluster PSP is restrictive. (https://github.com/openebs/openebs/issues/3331)
+- Fixed an issue where cStor pool migration from SPC to CSPC could result in pool failure when migration takes time and the user tries to restart the SPC pool. (https://github.com/openebs/upgrade/issues/82)
+- Fixed an issue with NDM partition discovery causing errors when partitions were present on top of filtered devices like OS disk. (https://github.com/openebs/openebs/issues/3321)
+ 
+
+### Backward Incompatibilities
+
+- Kubernetes 1.17 or higher release is recommended as this release contains the following updates that will not be compatible with older Kubernetes releases. 
+  * The CRD version has been upgraded to `v1`. (Thanks to the efforts from @RealHarshThakur, @prateekpandey14, @akhilerm)
+  * The CSI components have been upgraded to: 
+      * k8s.gcr.io/sig-storage/csi-attacher:v3.1.0
+      * k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.1.0
+      * k8s.gcr.io/sig-storage/csi-provisioner:v2.1.0
+      * k8s.gcr.io/sig-storage/csi-resizer:v1.1.0
+      * k8s.gcr.io/sig-storage/csi-snapshotter:v4.0.0
+      * k8s.gcr.io/sig-storage/snapshot-controller:v4.0.0
+      * k8s.gcr.io/sig-storage/csi-snapshotter:v3.0.3 (for cStor CSI volumes)
+      * k8s.gcr.io/sig-storage/snapshot-controller:v3.0.3 (for cStor CSI volumes)
+
+- If you are upgrading from an older version of cStor Operators to this version, you will need to manually delete the cStor CSI driver object prior to upgrading. `kubectl delete csidriver cstor.csi.openebs.io`. For complete details on how to upgrade your cStor Operators, see https://github.com/openebs/upgrade/blob/master/docs/upgrade.md#cspc-pools.
+
+- The CRD API version has been updated for the cStor custom resources to v1. If you are upgrading via the helm chart, you might have to make sure the new CRDs are updated. https://github.com/openebs/cstor-operators/tree/master/deploy/helm/charts/crds
```

Comment:
you might have to make sure the new CRDs are updated. -> you might have to make sure that the new CRDs are updated.